frontend/pages/admin/IntegrationsPage/cards/MdmSettings/WindowsMdmPage/WindowsMdmPage.tsx

```diff
@@ -113,17 +137,64 @@ const WindowsMdmPage = ({ router }: IWindowsMdmPageProps) => {
             onChange={onChangeMdmOn}
             disabled={gitOpsModeEnabled}
           />
-          <p>{descriptionText}</p>
-          <Checkbox
-            disabled={!isPremiumTier || !mdmOn || gitOpsModeEnabled}
-            value={autoMigration}
-            onChange={onChangeAutoMigration}
-            labelTooltipContent={
-              isPremiumTier ? "" : "This feature is included in Fleet Premium."
-            }
-          >
-            Automatically migrate hosts connected to another MDM solution
-          </Checkbox>
+          {!isPremiumTier && <p>{descriptionText}</p>}
+          {isPremiumTier && (
+            // NOTE: first time using fieldset and legend. if we use this more we should make
+            // a reusable component
+            <fieldset
+              disabled={!mdmOn}
+              className={`${baseClass}__enrollment-type-fieldset form-field`}
+            >
+              {/* NOTE: we use this wrapper div to style the legend since legend
+               does not work well with flexbox. the wrapper div helps the gap styling apply. */}
+              <div>
+                <legend className="form-field__label">
+                  End user experience
+                </legend>
+              </div>
+              <Radio
+                id="automatic-enrollment"
+                label="Automatic"
+                value="automaticEnrollment"
+                name="enrollmentType"
+                checked={enrollmentType === "automatic"}
+                onChange={onChangeEnrollmentType}
+                disabled={!mdmOn}
+                helpText="MDM is turned on when Fleet's agent is installed on Windows hosts (excluding servers)."
+              />
+              <Radio
+                id="manual-enrollment"
+                label="Manual"
+                value="manualEnrollment"
+                name="enrollmentType"
+                checked={enrollmentType === "manual"}
+                onChange={onChangeEnrollmentType}
+                disabled={!mdmOn}
+                helpText={
+                  <>
+                    Requires{" "}
+                    <CustomLink
+                      text="connecting Fleet to Microsoft Entra."
+                      url={
+                        PATHS.ADMIN_INTEGRATIONS_AUTOMATIC_ENROLLMENT_WINDOWS
+                      }
+                    />{" "}
+                    End users have to manually turn on MDM in{" "}
+                    <b>Settings &gt; Access work or school.</b>
+                  </>
+                }
+              />
+            </fieldset>
+          )}
+          {isPremiumTier && enrollmentType !== "manual" && (
```

Comment:
Isn't it possible for enrollmentType to be null here? Double-checking in that case, you want this checkbox to display, right?